server/app.js

```diff
@@ -43,6 +46,17 @@ app.use((err, req, res) => {
   res.status(err.status || 500);
   res.render('error');
 });
+mongoose.connect(process.env.DB_HOST, {
```

Comment:
Please try to add the new lines after every block.

Reply:
Ill do that